kpTranslator/controllers/query_controller.py

```diff
@@ -15,4 +21,23 @@ def query(request_body):  # noqa: E501
 
     :rtype: Message
     """
-    return 'do some magic!'
+    request = Query.from_dict(request_body)
+    query_graph = request.message.query_graph
+
+    qnodes = query_graph.nodes
+    qedges = query_graph.edges
```

Comment:
You're not doing anything with this value, what is it for?

Reply:
I've removed line 28 in the latest commit since I'm not using it for now. This initial query response code is rudimentary and will be expanded to more properly respond to queries.

Reply:
Removed for now since I'm not using it